plugins/confluence-plugin/src/components/PageContent.tsx

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import React, { useState, useEffect } from "react";
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+// import React from "react";
+import { isEmpty, isNil } from "lodash";
+import "../baseStyles.css";
+import {
+  Box,
+  Text,
+  usePluginContext,
+} from "@cortexapps/plugin-core/components";
+import { getEntityYaml } from "../api/Cortex";
+import { getConfluenceDetailsFromEntity } from "../lib/parseEntity";
+
+const baseJIRAUrl = "https://cortex-se-test.atlassian.net";
+
+const PageContent: React.FC = () => {
+  const [pageContent, setPageContent] = useState<string | undefined>();
+  const [pageTitle, setPageTitle] = useState<string | undefined>();
+  const [entityPage, setEntityPage] = useState<any | string>();
+
+  const context = usePluginContext();
+
+  useEffect(() => {
+    const fetchEntityYaml = async (): Promise<void> => {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      const entityTag = context.entity?.tag;
+      if (!isNil(entityTag)) {
+        try {
+          const yaml = await getEntityYaml(context.apiBaseUrl, entityTag);
+          const pageID = isEmpty(yaml)
+            ? undefined
+            : getConfluenceDetailsFromEntity(yaml);
+          setEntityPage(pageID?.pageID);
+          const jiraURL =
+            baseJIRAUrl +
+            "/wiki/rest/api/content/" +
+            // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+            pageID?.pageID +
+            "?expand=body.view";
+          const contentResult = await fetch(jiraURL);
+          const contentJSON = await contentResult.json();
+          setPageContent(contentJSON.body.view.value);
+          setPageTitle(contentJSON.title);
+        } catch (e) {
+          console.error("Error fetching Confluence page: ", e);
```

Comment:
I think a better UX would be to show an error in the HTML instead of having them see a blank screen and have to open up dev tools to parse what's happening.